src/v2/Apps/Order/Components/Mutations/useSetPayment.ts

```diff
@@ -0,0 +1,31 @@
+import { useMutation } from "v2/Utils/Hooks/useMutation"
+import { graphql } from "react-relay"
+import { useSetPaymentMutation } from "v2/__generated__/useSetPaymentMutation.graphql"
+
+export const useSetPayment = () => {
+  return useMutation<useSetPaymentMutation>({
+    mutation: graphql`
+      mutation useSetPaymentMutation($input: CommerceSetPaymentInput!) {
+        commerceSetPayment(input: $input) {
+          orderOrError {
+            ... on CommerceOrderWithMutationSuccess {
+              order {
+                id
+                creditCard {
+                  internalID
+                }
```

Comment:
Not sure if accessing credit card will cause errors but this can be removed now I believe.